### PR TITLE
LL-2135 New Countervalues module

### PR DESCRIPTION
**Pull request by JunichiSugiura**

### CODE REVIEW

**Comment by JunichiSugiura** [comment]
TODO
- [ ] Throttled polling

**Comment by gre** [comment]
Todo: switch to production API url

**Comment by gre** [comment]
TODO: remove completely the compound module of the poc, we no longer will need it.

File: src/countervalues/react.js [hunk 1]
```diff
@@ -91,18 +93,23 @@ export const Countervalues = ({
...
   useEffect(() => {
-    if (
-      JSON.stringify(userSettings) === JSON.stringify(userSettingsRef.current)
-    )
-      return;
+    if (isFirstRender) {
+      isFirstRender.current = false;
+    } else {
+      if (
+        JSON.stringify(userSettings) === JSON.stringify(userSettingsRef.current)
```
Comment (gre) [inline src/countervalues/react.js:102]: (LOW) I feel this is overkill to stringify each time, I wonder if there is a better way, but that was for sure the most MVP one.

**Comment by gre** [comment]
- 🙏 please bump on LLD & LLM
- on thursday we consider QA on it and I think we should be able to merge it 🙏

**Comment by JunichiSugiura** [comment]
`16.0.0-countervalues.0` isn't ready to be used in LLD/LLM yet.
It needs to wait till sell impl is merged.

**Review by gre — APPROVED** [review]

**Pull request merged**